tests/test_vpc.rb

```diff
@@ -21,12 +21,13 @@
 # TestVpc - Minitest for Vpc node utility class
 class TestVpc < CiscoTestCase
   def setup
+    skip("Test not supported on #{node.product_id}") if platform != :nexus
     super
     no_feature_vpc
   end
 
   def teardown
-    no_feature_vpc
+    no_feature_vpc if platform == :nexus
```

Comment:
this change won't be needed if you use `@skip_unless_supported` as suggested above